libraries/Microsoft.Bot.Builder.Classic/Microsoft.Bot.Builder.Classic/packages.config

```diff
@@ -2,7 +2,13 @@
 <packages>
   <package id="Autofac" version="3.5.2" targetFramework="net461" />
   <package id="Chronic.Signed" version="0.3.2" targetFramework="net461" />
+  <package id="Microsoft.Recognizers.Text" version="1.0.1.1" targetFramework="net461" />
+  <package id="Microsoft.Recognizers.Text.DateTime" version="1.0.1.1" targetFramework="net461" />
+  <package id="Microsoft.Recognizers.Text.Number" version="1.0.1.1" targetFramework="net461" />
+  <package id="Microsoft.Recognizers.Text.NumberWithUnit" version="1.0.1.1" targetFramework="net461" />
   <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net461" />
   <package id="Newtonsoft.Json" version="10.0.3" targetFramework="net461" />
   <package id="NuGet.CommandLine" version="4.5.1" targetFramework="net461" developmentDependency="true" />
+  <package id="System.Collections.Immutable" version="1.4.0" targetFramework="net461" />
```

Comment:
These packages should not be added either. Introducing more dependencies makes things more complicated.